pyproject.toml

```diff
@@ -3,6 +3,7 @@ requires = ["setuptools>=45", "setuptools_scm[toml]==7.*"]
 build-backend = "setuptools.build_meta"
 
 [project]
+requires-python = ">3.9.6"
```

Comment:
I thought we were on 3.7. What's the reasoning for this specific version?

Reply:
https://github.com/opengisch/django-ogcapif/blob/main/.github/workflows/publish.yml#L17 is the reason, but happy to change if you prefer